fhevm/operators_arithmetic.go

```diff
@@ -3,11 +3,190 @@ package fhevm
 import (
 	"encoding/hex"
 	"errors"
+	"math/big"
 
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/zama-ai/fhevm-go/sgx"
 	"go.opentelemetry.io/otel/trace"
 )
 
+func sgxAddRun(environment EVMEnvironment, caller common.Address, addr common.Address, input []byte, readOnly bool, runSpan trace.Span) ([]byte, error) {
```

Comment:
This file has almost 500 lines. Let's move all the `sgx*Run` functions into a separate file `sgx_arithmetic.go` in the same package